### PR TITLE
Added release PR commenter

### PR DESCRIPTION
**Pull request by goruha**
## what
* Added release PR commenter

### CODE REVIEW

File: .github/workflows/shared-release-branches.yml [hunk 1]
```diff
@@ -39,3 +37,175 @@ jobs:
...
+                issue_number: pr_id,
+                per_page: 100,
+              }
+
+              let existingComment = null
+            
+              const messageId = `<!-- release-pr-comment:${release.id} -->`
+              const message = `
+              ${messageId}
+              This functionality was released in [release.name](${release.html_url}).
```
Comment (osterman) [inline .github/workflows/shared-release-branches.yml:69]:
```suggestion
              This functionality was released in [${release.name}](${release.html_url}).
```

File: .github/workflows/shared-release-branches.yml [hunk 2]
```diff
@@ -39,3 +37,175 @@ jobs:
...
+                })
+              
+              // Initialize existingComment variable
+              let existingComment = null;
+              
+              // Constructing the message to be posted or updated as a comment
+              const messageId = `<!-- release-pr-comment:${release.id} -->`;
+              const message = `
+                ${messageId}
+                This PR is included in the release ${release.html_url}
```
Comment (osterman) [inline .github/workflows/shared-release-branches.yml:85]: Why two messages?

File: .github/workflows/shared-release-branches.yml [hunk 3]
```diff
@@ -39,3 +37,175 @@ jobs:
...
+                issue_number: pr_id,
+                per_page: 100,
+              }
+
+              let existingComment = null
+            
+              const messageId = `<!-- release-pr-comment:${release.id} -->`
+              const message = `
+              ${messageId}
+              This functionality was released in [${release.name}](${release.html_url}).
```
Comment (osterman) [inline .github/workflows/shared-release-branches.yml:69]:
```suggestion
              These changes were included as part of the [${release.name}](${release.html_url}) release.
```

Comment (goruha) [reply .github/workflows/shared-release-branches.yml]: Done

File: .github/workflows/shared-release-branches.yml (hunk 2 above)
Comment (goruha) [reply .github/workflows/shared-release-branches.yml]: Fixed

File: .github/workflows/shared-release-branches.yml [hunk 4]
```diff
@@ -39,3 +37,162 @@ jobs:
...
+              }
+              
+              // Initialize existingComment variable
+              let existingComment = null;
+              
+              // Constructing the message to be posted or updated as a comment
+              const messageId = `<!-- release-pr-comment:${release.id} -->`;
+              const message = `
+                ${messageId}
+                These changes were included as part of the [${release.name}](${release.html_url}) release.
```
Comment (osterman) [inline .github/workflows/shared-release-branches.yml:71]:
```suggestion
                These changes were released in [${release.name}](${release.html_url}).
```

**Review by osterman — APPROVED** [review]

**Pull request merged**